test/db/cmd/cmd_ag

```diff
@@ -0,0 +1,337 @@
+NAME=agi
+FILE=bins/elf/hello_world
+CMDS=<<EOF
+aa 2> /dev/null
+agi*
+EOF
+EXPECT=<<EOF
+agn "sym.imp.free" base64:
```

Comment:
master currently doesn't print the empty string. Which, according to the help of `agn` seems wrong because a body is expected. However currently `agn <title> base64:` fails.

Reply:
I hav

Reply:
No problems removing it. The behavior in current master is a bit inconsistent. If you do `agi*; .agi*; agg*` first one prints without `base64:` but last one prints it.